actors/runtime/src/main/java/cloud/orbit/actors/runtime/SimpleInvocationHandler.java

```diff
@@ -90,6 +90,7 @@
 
         // Perform the internal invocation
         final Task<Object> invokeResult = doInvoke(runtime, invocation, entry, target, method, reentrant, invoker);
+        invokeResult.putMetadata("methodName", method.getName());
```

Comment:
Not sure where to put this... it would be nice if it wasn't in the framework itself.